src/hyp3_isce2/burst.py

```diff
@@ -262,7 +214,19 @@ def spoof_safe(burst: BurstMetadata, burst_tiff_path: Path, base_path: Path = Pa
     return safe_path
 
 
-def get_region_of_interest(poly1: geometry.Polygon, poly2: geometry.Polygon, is_ascending: bool = True) -> Tuple[float]:
+def get_isce2_burst_bbox(params: BurstParams, base_dir: Path = Path.cwd()) -> geometry.Polygon:
+    s1_obj = Sentinel1()
+    s1_obj.configure()
+    s1_obj.safe = [str(base_dir / f'{params.granule}.SAFE')]
+    s1_obj.swathNumber = int(params.swath[-1])
+    s1_obj.parse()
+    snwe = s1_obj.product.bursts[params.burst_number].getBbox()
+    # convert from south, north, west, east -> minx, miny, maxx, maxy
+    bbox = geometry.box(snwe[2], snwe[0], snwe[3], snwe[1])
```

Comment:
For some reason I thought South was generally equivalent to max Y? We should talk about this to clear up my confusion.

Reply:
No worries, South should be `miny`. When I get confused, or just want to check something, I go to vertex and move my cursor around the map. It displays your latitude/longitude in the upper left corner so you can see how these values change as you move.